event_registration_partner_unique/models/event.py

```diff
@@ -21,31 +19,3 @@ def _check_forbid_duplicates(self):
         return self.filtered(
             "forbid_duplicates"
         ).registration_ids._check_forbid_duplicates()
-
```

Comment:
Don't split this into a separate file. It's OK in this one, as it's a children model (like `sale.order` with `sale.order.line`. See the exception in the guidelines: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming